### ISSUE TITLE
Graphic Filter video lag (all filters)

### ISSUE THREAD

**Issue by kristianity77** [TheOfficialFloW/Adrenaline#278]
Genuine issue here but not sure if its just the price you pay for graphical filters.

There is definite lag on any filter, which is obviously more noticeable on games the require timing.  So any game like Hot Shots Golf, Darius Burst, Patapon, DJ Max etc its more noticeable than others.  But it is there on all games, its not title specific.

Its not controller lag thats introduced as this is easily shown with a game like Hot Shots Golf when taking a shot.  A button press registers with a click which is instant in terms of sound.  But the image on the screen with filters on is about 4 or 5 frames behind what should be displayed.  So you always miss shots as a late press.  Turning the filters off and the game is perfect with no lag anywhere.

Basically it seems that the graphical filters added by adrenaline, is adding about a tenth of a second lag to the image being displayed.  Is this the way it has to be or can it be rectified.  Its a shame either way right now because the filters are really really good otherwise!
